common/src/main/scala/org/apache/spark/sql/comet/util/Utils.scala

```diff
@@ -264,4 +265,24 @@ object Utils {
         throw new SparkException(s"Unsupported Arrow Vector for $reason: ${valueVector.getClass}")
     }
   }
+
+  /**
+   * Imports data from ArrowArray/ArrowSchema into a FieldVector. This is basically the same as
+   * Java Arrow `Data.importVector`. `Data.importVector` initiates `SchemaImporter` internally
+   * which is used to fill dictionary ids for dictionary encoded vectors. Every call to
+   * `importVector` will begin with dictionary ids starting from 0. So, separate calls to
+   * `importVector` will overwrite dictionary ids. To avoid this, we need to use the same
+   * `SchemaImporter` instance for all calls to `importVector`.
+   */
+  def importVector(
```

Comment:
ditto: may be it's better to put this in a separate class in Java other than spark.sql.comet.util?

Reply:
Okay